example_analysis/0.configure_preprocess_params.ipynb

```diff
@@ -20,7 +20,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    !pwd
</pre><p>Can this be removed now that testing is done?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cheeseman-lab/brieflow/pull/23/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>